ipn/ipnlocal/profiles.go

```diff
@@ -174,9 +174,13 @@ func (pm *profileManager) SetPrefs(prefsIn ipn.PrefsView) error {
 		}
 		cp.LocalUserID = pm.currentUserID
 	}
+	if prefs.ProfileName() != "" {
```

Comment:
Do we want to enforce uniqueness for names?

Reply:
Good point.

Reply:
Done. PTAL